domdiv/draw.py

```diff
@@ -1075,64 +1426,260 @@ def drawSetNames(self, pageCards):
         finally:
             self.canvas.restoreState()
 
-    def drawDividers(self, cards):
-        # split into pages
-        cards = split(cards, self.options.numDividersVertical *
-                      self.options.numDividersHorizontal)
+    def calculatePages(self, cards):
+        options = self.options
 
-        # Starting with tabs on the left or the right?
-        if self.options.tab_side in ["right-alternate", "right"]:
-            self.odd = True
+        # Adjust for Vertical vs Horizontal
+        if options.orientation == "vertical":
+            options.dividerWidth, options.dividerBaseHeight = options.dominionCardHeight, options.dominionCardWidth
         else:
-            # left-alternate, left, full
-            self.odd = False
-
-        for pageNum, pageCards in enumerate(cards):
-            # remember whether we start with odd or even divider for tab
-            # location
-            pageStartOdd = self.odd
+            options.dividerWidth, options.dividerBaseHeight = options.dominionCardWidth, options.dominionCardHeight
+
+        options.fixedMargins = False
+
+        if options.tabs_only:
+            # fixed for Avery 8867 for now
```

Comment:
Looks like there's some old code being reintroduced here?

Reply:
Yes, I missed this in the merge since the code changed locations.